modules/RTC/RTCBrowserType.js

```diff
@@ -226,7 +226,7 @@ const RTCBrowserType = {
         // This mirrors what sdp-simulcast uses (which is used when deciding
         // whether to actually enable simulcast or not).
         // TODO: the logic should be in one single place.
-        return window.chrome !== undefined;
+        return window.chrome !== undefined && !RTCBrowserType.isEdge();
```

Comment:
I'm afraid. I see `isSimulcastSupported()` and `supportsSimulcast()` (which still checks `window.chrome`) in RTCBrowserType. Help please.